lib/robots/dor_repo/speech_to_text/stt_create.rb

```diff
@@ -11,10 +11,51 @@ def initialize
 
         # available from LyberCore::Robot: druid, bare_druid, workflow_service, object_client, cocina_object, logger
         def perform_work
-          # TODO: do the caption creation by calling out to Whisper
+          # start the caption creation by sending an SQS message to the caption creation service
+
+          send_sqs_message
 
           # Leave this step running until the Whisper monitoring job marks it as complete
           LyberCore::ReturnState.new(status: :noop, note: 'Initiated SpeechToText.')
+        rescue StandardError => e
+          Honeybadger.notify('Problem sending SQS Message to AWS for SpeechToText', context: { druid:, error: e })
+          raise "Error sending SQS message: #{e.message}"
+        end
+
+        private
+
+        def aws_provider
+          @aws_provider ||= Dor::TextExtraction::AwsProvider.new(region: Settings.aws.region, access_key_id: Settings.aws.access_key_id, secret_access_key: Settings.aws.secret_access_key)
+        end
```

Comment:
i wonder if there is a nicer way to make this `aws_provider` available to any robot that needs it (currently two) without having to reinitialize like this each time (kind of like how we make the workflow_service client available to all robots)

Reply:
Pushed a commit to extract this a new base robot class that we can subclass from.  Not sure how much of an improvement it is other than reducing some duplicated code